src/mds/topology/topology_metric.h

```diff
@@ -238,8 +238,34 @@ struct LogicalPoolMetric {
 };
 using LogicalPoolMetricPtr = std::unique_ptr<LogicalPoolMetric>;
 
+struct ClusterMetric {
```

Comment:
The poolNum, serverNum and chunkserverNum of cluster is also needed.

Reply:
fix

Reply:
The serverNum is missed.